src/components/widgets/Clock.js

```diff
@@ -7,25 +9,19 @@ import Header from './Header'
 class Clock extends Component {
   constructor( props ) {
     super( props )
-    this.state = {
-      time: null,
-      amPm: null,
-      dayOfWeek: null,
-      date: null
-    }
 
     this.setDateAndTime = this.setDateAndTime.bind(this)
   }
 
   setDateAndTime() {
```

Comment:
We can get rid of this function now (this data will be set during the reduction).  We need to update `componentDidMount` to call the dispatch method for the clock action.